src/gpu/DNNMark/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: GCN3 DNNMark Tests
+title: VEGA DNNMark Tests
```

Comment:
Same thing with many of these files.  Did you include a patch from Matt P or something?